lib/post_request_task/tests.py

```diff
@@ -0,0 +1,88 @@
+from django.core.signals import request_finished, request_started
+from django.test import TestCase
+
+from mock import Mock, patch
+from nose.tools import eq_
+
+from lib.post_request_task.task import (task, _get_task_queue, _discard_tasks,
+                                        _stop_queuing_tasks)
+
+
+task_mock = Mock()
+
+
+@task
+def test_task():
+    task_mock()
+
+
+class TestTask(TestCase):
+
+    def setUp(self):
+        super(TestTask, self).setUp()
+        task_mock.reset_mock()
+        _discard_tasks()
+
+    def tearDown(self):
+        task_mock.reset_mock()
+        _discard_tasks()
+        _stop_queuing_tasks()
+        super(TestTask, self).tearDown()
+
+    def _verify_task_filled(self):
```

Comment:
Renaming this `_verify_one_task_queued()` would make the code calling this function less ambiguous.